osxcollector/osxcollector.py

```diff
@@ -913,6 +915,7 @@ def collect(self, section_list=None):
             ('safari', self._collect_safari),
             ('accounts', self._collect_accounts),
             ('mail', self._collect_mail),
+            ('executables', self._collect_binary_names_in_path)
```

Comment:
Is there a missing `,` at the end of this line?

Reply:
++ I missed this copy pasting between different checkouts